src/test/java/woohakdong/server/api/service/auth/AuthServiceTest.java

```diff
@@ -0,0 +1,57 @@
+package woohakdong.server.api.service.auth;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import woohakdong.server.common.exception.CustomException;
+import woohakdong.server.domain.member.MemberRepository;
+import woohakdong.server.domain.school.School;
+import woohakdong.server.domain.school.SchoolRepository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+import static woohakdong.server.common.exception.CustomErrorInfo.*;
+
+@ActiveProfiles("test")
+@SpringBootTest
+class AuthServiceTest {
+
+    @Autowired
+    private MemberRepository memberRepository;
+
+    @Autowired
+    private SchoolRepository schoolRepository;
+
+    @Autowired
+    private AuthService authService;
+
+    @DisplayName("학교 도메인인 구글 이메일로만 로그인 할 수 있다.")
+    @Test
+    void schoolDomainValidation() {
+        // given
+        String email = "uiyeop@ajou.ac.kr";
+
+        //when
+        School school = authService.checkSchoolDomain(email);
+
+        //then
+        assertThat(school).isNotNull();
+        assertThat(school.getSchoolDomain()).isEqualTo("ajou.ac.kr");
+
+
+    }
+
+    @DisplayName("학교 도메인이 아닌 구글 이메일로는 로그인 할 수 없다.")
+    @Test
+    void schoolDomainInValidation() {
+        // given
+        String email = "uiyeop@gmail.com";
+
+        //when & then
+        CustomException exception = assertThrows(CustomException.class, () -> authService.checkSchoolDomain(email));
```

Comment:
실패 케이스 테스트 코드 작성해주신 것 너무 좋은 것 같습니다.

```suggestion
        import static org.assertj.core.api.Assertions.assertThatThrownBy;

        assertThatThrownBy(() -> authService.checkSchoolDomain(email))
                .isInstanceOf(CustomException.class)
                .hasMessage(INVALID_SCHOOL_DOMAIN.getMessage());
```
위처럼 작성하는게 좋을 것 같습니다.
assertj의 Assertions.assertThat 메소드를 사용하기로 결정한 이상, 예외 처리도 assertj의 메소드를 사용하는게 일관성 있게 좋을 것 같습니다.